src/main/java/com/jcabi/aspects/aj/ImmutabilityChecker.java

```diff
@@ -73,8 +74,10 @@ public void after(final JoinPoint point) {
             throw new IllegalStateException(
                 String.format(
                     // @checkstyle LineLength (1 line)
-                    "%s is not immutable, can't use it (jcabi-aspects ${project.version}/${buildNumber})",
-                    type
+                    "%s is not immutable, can't use it (jcabi-aspects %s/%s)",
```

Comment:
No, and judging by the fact that is wasn't covered by a test when I started working on this issue (see [my comment](https://github.com/jcabi/jcabi-aspects/issues/167#issuecomment-119674528)), it isn't yet supposed to be. I think covering it with a test would be a separate issue.